scripts/install-systemd-multi-user.sh

```diff
@@ -9,6 +9,39 @@ readonly SERVICE_DEST=/etc/systemd/system/nix-daemon.service
 readonly SOCKET_SRC=/lib/systemd/system/nix-daemon.socket
 readonly SOCKET_DEST=/etc/systemd/system/nix-daemon.socket
 
+
+# Path for the systemd override unit file to contain the proxy settings
+readonly SERVICE_OVERRIDE=${SERVICE_DEST}.d/override.conf
+
+create_systemd_override() {
+     header "Start: configure proxy for the nix-daemon service"
+    _sudo "create directory for systemd unit override" mkdir -p $(dirname $SERVICE_OVERRIDE)
```

Comment:
```

In /build/install-systemd-multi-user.sh line 18:
    _sudo "create directory for systemd unit override" mkdir -p $(dirname $SERVICE_OVERRIDE)
                                                                ^--------------------------^ SC2046: Quote this to prevent word splitting.
```

```suggestion
    _sudo "create directory for systemd unit override" mkdir -p "$(dirname $SERVICE_OVERRIDE)"
```